.github/unittest.yml

```diff
@@ -0,0 +1,29 @@
+name: Unittest xcube-clms
```

Comment:
So far the CI is not executed. I think it is because the unittest.yml needs to be in a folder called workflows. Please double check with other xcube plugin or xcube itself.